src/SHL-0.5.xml

```diff
@@ -0,0 +1,109 @@
+<SPDXLicenseCollection xmlns="http://www.spdx.org/license">
+    <license isOsiApproved="false" licenseId="Solderpad-0.5" name="Solderpad Hardware License v0.5">
```

Comment:
"Solderpad-0.5" is listed here as the licenseId, but the filename is "SHL-0.5" so it can't find the corresponding test file. I'm not sure which one is the preferred ID, but I believe that's why the test is failing so you'll want these to be aligned.